lua/entities/wired_door/init.lua

```diff
@@ -203,18 +217,7 @@ function ENT:makedoor(ply,trace,ang,model,open,close,autoclose,closetime,class,h
 
 	self.xent = entit
 	self.xclass = tostring(class)
-end
 
-function switch(t)
-  t.case = function (self,x)
-    local f=self[x] or self.default
-    if f then
-      if type(f)=="function" then
-        f(x,self)
-      else
-        error("case "..tostring(x).." not a function")
-      end
-    end
-  end
-  return t
+	return entit

```

Comment:
Diff is messed up. The function that creates the door entity now returns the door entity so we can set undo/ownership in the tool